chatbot/ai-studio.yaml

```diff
@@ -0,0 +1,27 @@
+application:
+  type: language
+  name: chatbot
+  description: This is a LLM chatbot application that can interact with a llamacpp model-service
+  containers:
+    - name: chatbot-inference-app
+      contextdir: ai_applications
+      containerfile: builds/Containerfile
```

Comment:
the chatbot-inference-app also should include arm64 && amd64 arch

Reply:
done 👍